src/commands/Utilities/application.ts

```diff
@@ -0,0 +1,49 @@
+import { FireMessage } from "@fire/lib/extensions/message";
+import { MessageEmbed, Permissions } from "discord.js";
+import {APIApplication} from 'discord-api-types'
+import { Language } from "@fire/lib/util/language";
+import { Command } from "@fire/lib/util/command";
+
+export default class Application extends Command {
+  constructor() {
+    super("application", {
+      description: (language: Language) =>
+        language.get("APPLICATION_COMMAND_DESCRIPTION"),
+      clientPermissions: [
+        Permissions.FLAGS.SEND_MESSAGES,
+        Permissions.FLAGS.EMBED_LINKS,
+      ],
+      args: [
+        {
+          id: "application",
+          type: "string",
+          default: undefined,
+          required: true,
+        },
+      ],
+      aliases: ["applicationinfo", "infoapplication", "appinfo", "infoapp", "app"],
+      enableSlashCommand: true,
+      restrictTo: "all",
+      slashOnly: true,
+    });
+  }
+
+  async exec(
+    message: FireMessage,
+  {id}: { id: string }
+  ) {
+    const application: APIApplication = await this.client.req.applications(id).rpc.get().catch(() => {});
```

Comment:
```suggestion
    const application: APIApplication = await this.client.req.applications(id).rpc.get().catch(() => null);
```
cuz
![image](https://user-images.githubusercontent.com/68508885/188334412-b794f5c2-b00b-4751-8591-8ad62dc12a37.png)

Reply:
nvm ig just noticed this is almost year old pr lmao